console/database/saved_filters_test.go

```diff
@@ -98,6 +99,58 @@ func TestSavedFilter(t *testing.T) {
 	}
 }
 
+func TestSavedFilterSqlite(t *testing.T) {
+	r := reporter.NewMock(t)
+
+	testSavedFilter(t, NewMock(t, r, DefaultConfiguration()))
+}
+
+func TestSavedFilterPostgres(t *testing.T) {
+	server := helpers.CheckExternalService(t, "Postgresql", []string{"postgres:5432", "127.0.0.1:5432"})
+
+	server, serverPort, err := net.SplitHostPort(server)
+
+	if err != nil {
+		t.Fatalf("failed to parse server: %s", err)
+	}
+
+	r := reporter.NewMock(t)
+
+	dsn := fmt.Sprintf("host=%s port=%s user=akvorado password=akpass dbname=akvorado sslmode=disable", server, serverPort)
+	c := NewMock(
+		t,
+		r,
+		Configuration{
+			Driver: "postgresql",
+			DSN:   dsn,
+		},
+	)
+
+	testSavedFilter(t, c)
+
+	// clean database for future tests
+	db, err := sql.Open("pgx", dsn)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Unable to connect to database: %v\n", err)
+		os.Exit(1)
+	}
+	defer db.Close()
+	if _, err := db.Exec(`
+	DO $$ DECLARE
+    r RECORD;
+BEGIN
+    -- if the schema you operate on is not "current", you will want to
+    -- replace current_schema() in query with 'schematodeletetablesfrom'
+    -- *and* update the generate 'DROP...' accordingly.
+    FOR r IN (SELECT tablename FROM pg_tables WHERE schemaname = current_schema()) LOOP
+        EXECUTE 'DROP TABLE IF EXISTS ' || quote_ident(r.tablename) || ' CASCADE';
+    END LOOP;
+END $$;
+`); err != nil {
+		t.Fatalf("failed to clean database: %s", err)
+	}
+}
```

Comment:
You should put that as a function called from `t.Cleanup()` to ensure it is still  triggered if there is a test failure.